crates/router/src/configs/defaults.rs

```diff
@@ -4474,14 +4474,249 @@ impl Default for super::settings::RequiredFields {
                         enums::PaymentMethodType::Eps,
                         ConnectorFields {
                             fields: HashMap::from([
+                                (
+                                    enums::Connector::Adyen,
+                                    RequiredFieldFinal {
+                                        mandate: HashMap::new(),
+                                        non_mandate: HashMap::new(),
+                                        common: HashMap::from([
+                                            (
+                                                "payment_method_data.bank_redirect.eps.bank_name".to_string(),
+                                                RequiredFieldInfo {
+                                                    required_field: "payment_method_data.bank_redirect.eps.bank_name".to_string(),
+                                                    display_name: "bank_name".to_string(),
+                                                    field_type: enums::FieldType::UserBank,
+                                                    value: None,
+                                                }
+                                            )
+                                        ]),
+                                    }
+                                ),
                                 (
                                     enums::Connector::Stripe,
+                                    RequiredFieldFinal {
+                                        mandate: HashMap::new(),
+                                        non_mandate: HashMap::from([
+                                            (
+                                                "payment_method_data.bank_redirect.eps.billing_details.billing_name".to_string(),
```

Comment:
here we made billing_name as mandatory field but in the transformers it is handled as option field?

Reply:
Made `billing_name` mandatory in the transformer too

Reply:
but was this necessary? does stripe consider the billing name as mandatory field?

Reply:
yes stripe considers it as a required field